lms/static/js/spec/edxnotes/views/shim_spec.js

```diff
@@ -233,5 +233,25 @@ define([
                 expect(mockViewer.element.appendTo).toHaveBeenCalledWith(annotators[0].wrapper);
             });
         });
+
+        describe('TagsPlugin', function () {
```

Comment:
Note: I was not able to get a unit test for the screen reader label working reliably (there also is not one for the text field screen reader label). Instead I added a bok choy test.